lib/commands/kendoui-notes.ts

```diff
@@ -0,0 +1,44 @@
+///<reference path="../.d.ts"/>
+"use strict";
+import {KendoUIBaseCommand} from "./kendoui-base";
+
+class KendoUINotesCommand extends KendoUIBaseCommand implements ICommand {
+
+	constructor(private $logger: ILogger,
+		private $opener: IOpener,
+		private $prompter: IPrompter,
+		$errors: IErrors,
+		$kendoUIService: IKendoUIService,
+		$loginManager: ILoginManager,
+		$options: IOptions,
+		$project: Project.IProject) {
+
+		super($errors, $project, $kendoUIService, $loginManager, $options);
+	}
+
+	allowedParameters: ICommandParameter[] = [];
+
+	execute(args: string[]): IFuture<void> {
+		return (() => {
+			let packages = _.filter(this.getKendoPackages().wait(), p => p.HasReleaseNotes);
+			this.$logger.out("You can review release notes for the following Kendo UI packages.");
+			this.$logger.out(this.$kendoUIService.getKendoPackagesAsTable(packages));
+			let schema: IPromptSchema = {
+				type: "input",
+				name: "packageIdx",
+				message: "Enter the index of the package",
+				validate: (value: string) => {
+					let num = parseInt(value, 10);
+					return !isNaN(num) && num >= 1 && num <= packages.length ? true : "Valid values are between 1 and " + packages.length;
+				}
+			};
+
+			let choice = this.$prompter.get([schema]).wait(),
```

Comment:
Ultimately it does work if there's only one package, so runnint
`$ appbuilder kendoui notes --latest --core` 
would work in a non-interactive console